### ISSUE TITLE
Question about the additional model import

### ISSUE THREAD

**Issue by dogyoonlee** [google-research/multinerf#127]
Hello,

I have a few questions about the Jax environment to develop my work.

Is it possible to construct the model such as some kernel in front of the nerf, which is flexibly applied depending the iteration?
For example, if I want to run the model which consists of multiple MLP after specific MultiNeRF training iteration, is it possible?
As I understand the Jax, the variable, which can be changed during training, can not be used as condition in if statement in the model.

In summary, 
How can I implement the additional kernel in front of the NeRF, which is started to train after the specific NeRF training iteration? 
Is it possible in Jax?

Thank you for your attention.
